index.d.ts

```diff
@@ -68,5 +69,6 @@ declare module 'react-native-document-picker' {
       options: DocumentPickerOptions<OS>
     ): Promise<DocumentPickerResponse[]>;
     static isCancel<IError extends { code?: string }>(err?: IError): boolean;
+    static releaseSecureAccess(uri: string): void;
```

Comment:
question: would it not be more convenient for this method to accept an array of uris?

Reply:
Good suggestion!